### PR TITLE
NEXUS-22094 - It is impossible to fetch custom package from group rep…

### PR DESCRIPTION
**Pull request by aornatovskyy**
Ticket: https://issues.sonatype.org/browse/NEXUS-22094

### CODE REVIEW

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RCommonHandlers.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    extends ComponentSupport
+{
+  /**
+   * Handle request of currently not supported metadata
+   *
+   * Related to:  NEXUS-22119
+   */
+  final Handler notSupportedMetadataRequest =
+      context -> HttpResponses.notFound("This metadata type is not supported for now.");
+}
```
Comment (chernikovmaksym) [inline nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RCommonHandlers.java]: new line

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy [hunk 2]
```diff
@@ -142,6 +142,19 @@ abstract class RRecipeSupport
         ))
   }
 
+  /**
+   * Matcher for all PACKAGES.rds metadata mapping.
```
Comment (chernikovmaksym) [inline nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy]: remove "all"

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy [hunk 3]
```diff
@@ -217,6 +230,15 @@ abstract class RRecipeSupport
     return new TokenMatcher('/{path:.+}/PACKAGES{extension:.*}')
   }
 
+  /**
+   * Token matcher for all PACKAGES.rds files.
```
Comment (chernikovmaksym) [inline nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy]: Same as above

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RCommonHandlers.java (hunk 1 above)
Comment (aornatovskyy) [reply nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RCommonHandlers.java]: c78bb7855bae1907a2ce78f436dfa0270a536d97

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy (hunk 2 above)
Comment (aornatovskyy) [reply nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy]: c78bb7855bae1907a2ce78f436dfa0270a536d97

File: nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy (hunk 3 above)
Comment (aornatovskyy) [reply nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy]: c78bb7855bae1907a2ce78f436dfa0270a536d97

**Pull request merged**